### DISCUSSION TITLE
Operating notes and possible bugs

### DISCUSSION THREAD

**Discussion by rpratt20** [mikaelnousiainen/RS41ng#73] [General]
I have noted a few things about DFM17 operation.  
      The power switch setting for the most part does not function to turn the sonde OFF.  It will stay OFF with batteries installed until turned ON.  Once ON I seldom get an OFF unless removing a battery.
    When selecting NMEA gps serial output, the transmitter remains transmitting. However no Lat/Lon is transmitted.  I have not been able to see what appears on the cable port as I do not have correct cables at this time. I would expect transmission to go off when tracking information is diverted from the radio.
POSSIBLE BUG:
      I am experiencing frequency changes of about 380000 hz for short periods.  The frequency changes for about 30 seconds and then returns to the stable frequency.  CW and Horus V2 appear on either frequency and return to stable frequency can also occur mid packet.  I have tried continuous Horus-V2, alternating CW with Horus-V2 and with a 10 second after transmit delay.  Only noticed on 1 of 3 sondes so far and problem began after setting NMEA output but does not correct with any further changes including back to starting point. ALSO NOTE: When the unexpected frequency is received Horus-v2 will decode as lower sideband.

**Comment by rpratt20** [comment]
I need to update on the possible bug.  I now think it is SDR# on Windows 11 that is the bug. In an effort to fix this I tried a different frequency.  The SDR would step through about 800 HZ and not change until making a jump.  Rebooting the laptop helped  but running the SDR on an old Windows XP computer corrected everything I was noting wrong.

**Comment by mikaelnousiainen** [comment]
@rpratt20 Thanks for testing the firmware on DFM-17! We will (slowly) go through the bug reports and try to fix remaining issues during upcoming months.

Please note that NMEA output is not (yet) supported on DFM-17 and might cause instability.
